internal/pkg/service/buffer/store/model/conditions.go

```diff
@@ -30,14 +30,17 @@ func (c ImportConditions) ShouldImport(s CurrentImportState) (bool, string) {
 	}
 
 	if s.Count >= c.Count {
-		return true, fmt.Sprintf("import count limit met, received: %d rows, limit: %d rows", s.Count, c.Count)
+		return true, fmt.Sprintf("import count threshold met, received: %d rows, threshold: %d rows", s.Count, c.Count)
 	}
 	if s.Size >= c.Size {
-		return true, fmt.Sprintf("import size limit met, received: %s, limit: %s", s.Size.String(), c.Size.String())
+		return true, fmt.Sprintf("import size threshold met, received: %s, threshold: %s", s.Size.String(), c.Size.String())
 	}
-	sinceLastImport := s.Now.Sub(s.LastImportAt).Truncate(time.Second)
+
+	now := time.Time(s.Now)
+	lastImportAt := time.Time(s.LastImportAt)
+	sinceLastImport := now.Sub(lastImportAt).Truncate(time.Second)
 	if sinceLastImport >= c.Time {
-		return true, fmt.Sprintf("import time limit met, last import at: %s, passed: %s limit: %s", s.LastImportAt.Format(time.Stamp), sinceLastImport.String(), c.Time.String())
+		return true, fmt.Sprintf("import time threshold met, last import at: %s, passed: %s threshold: %s", lastImportAt.Format(TimeFormat), sinceLastImport.String(), c.Time.String())
```

Comment:
Tu navrhujem:
- pouzit `threshold` namiesto `limit`, IMHO to ma trocha opacny vyznam (aspon X/maximalne X)
- rovnaky time format ako na ostatnych miestach